src/adhocracy_core/adhocracy_core/scripts/ad_auditlog.py

```diff
@@ -0,0 +1,85 @@
+"""Script to view auditlog entries."""
+
+import transaction
+import argparse
+import inspect
+import logging
+import pprint
+
+from datetime import datetime
+from pyramid.paster import bootstrap
+from substanced.interfaces import IRoot
+from substanced.util import get_auditlog
+
+from adhocracy_core.interfaces import AuditlogEntry
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():  # pragma: no cover
+    """View auditlog entries.
+
+    Show the auditlog entries for a time period:
+
+    ./bin/ad_auditlog -s 2016-01-01 -e 2016-02-1 etc/development.ini
+
+    If no startdate is given the start of the auditlog is used.
+    If no enddate is given the end of the auditlog is uded,
+
+    Additionally filtering by resource path is possible:
+    ./bin/ad_auditlog -s 2016-01-01 -e 2016-02-1 -p /path etc/development.ini
+    """
+    docstring = inspect.getdoc(main)
+    parser = argparse.ArgumentParser(description=docstring)
+    parser.add_argument('ini_file',
+                        help='path to the adhocracy backend ini file')
```

Comment:
The better place for argument descriptions is its help text instead of the general docstring.
I suggest to not add using examples and argument  descriptions to the docstring to remove duplication and ease refactoring (renaming things).